queries.sql

```diff
@@ -109,5 +109,59 @@ SELECT O.full_name, COUNT(A."name") as owned
 FROM owners O
 INNER JOIN animals A
 ON A.owners_id = O.id
-GROUP BY O.full_name ORDER BY owned DESC;
-
+GROUP BY O.full_name ORDER BY owned DESC
+LiMIT 1;
+
+SELECT A."name", MAX(VI.date) AS date FROM visits VI
+JOIN vets VE
+ON VE.id = VI.vets_id
+JOIN animals A
+ON VI.animals_id = A.id
+WHERE VE."name" = 'William Tatcher'
+GROUP BY A."name" ORDER BY date DESC
+LIMIT 1;
+
+SELECT COUNT(*) AS count FROM visits VI
+JOIN vets VE
+ON VE.id = VI.vets_id
+WHERE VE."name" = 'Stephanie Mendez';
+
+SELECT A."name", visits.date FROM visits
+JOIN animals A
+ON visits.animals_id = A.id
+WHERE visits.date >= '2020-04-01' AND visits.date <= '2020-08-30'
+ORDER BY visits.date DESC;
+
+SELECT A."name", COUNT(visits.date) as number_visits FROM visits
+JOIN animals A
+ON visits.animals_id = A.id
+GROUP BY A."name" ORDER BY number_visits DESC
+LIMIT 1;
+
+SELECT vets."name" as vets_name, S."name" as vet_specialization
+FROM vets
+FULL JOIN specializations SP
+ON vets.id = SP.vets_id
+FULL JOIN species S 
+ON SP.species_id = S.id;
+
+SELECT VE."name", COUNT(*) as not_specialities FROM visits
+JOIN animals A
+ON visits.animals_id = A.id
+JOIN species
+ON A.species_id = species.id
+JOIN vets VE
+ON visits.vets_id = VE.id
+JOIN specializations as S
+ON S.vets_id = visits.vets_id
+WHERE species.id != S.species_id
+GROUP BY VE."name" ORDER BY not_specialities DESC;
+
+SELECT A."name", visits.date as date FROM visits
+JOIN animals A
+ON visits.animals_id = A.id
+JOIN vets VE
+ON visits.vets_id = VE.id
+WHERE VE."name" = 'Maisy Smith'
+ORDER BY date
+LIMIT 1;
```

Comment:
- [ ] I have double-checked to confirm this. As requested by the [previous reviewer](https://github.com/aizjicod/vet-clinic-database/pull/4#discussion_r972561695), kindly include the query that says "What specialty should Maisy Smith consider getting?"
- [ ] Also, I have not seen a screenshot for this specific query. Kindly make sure that you include it in your pull request description.

![image](https://user-images.githubusercontent.com/86318284/190554587-270b2fe9-3b7c-42c1-93d4-86c632a012f7.png)

**I figured out this might have been a challenge for you and put the example query below:**


```
-- What specialty should Maisy Smith consider getting? Look for the species she gets the most
SELECT (species.name) AS speciesName, COUNT(species.id)
FROM animals
FULL OUTER JOIN species ON animals.species_id = species.id
JOIN visits ON animals.id = visits.animals_id
WHERE visits.vets_id = 2
GROUP BY species.name
ORDER BY COUNT DESC
LIMIT 1;
```